backend/pkg/helm/release.go

```diff
@@ -437,8 +455,13 @@ func (h *Handler) InstallRelease(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	err = h.setStatus("install", req.Name, processing, nil)
+	if err != nil {
+		zlog.Error().Err(err).Str("action", "install").Msg("failed to set status")
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+	}
+
```

Comment:
```suggestion
if err = h.setStatus("install", req.Name, processing, nil); err != nil {
		zlog.Error().Err(err).Str("action", "install").Msg("setting status")
		http.Error(w, err.Error(), http.StatusInternalServerError)
	}
```

Reply:
updated the error message but didn't change the if block style, as this style is used across.